src/language-js/utils.js

```diff
@@ -1473,6 +1421,46 @@ function hasIgnoreComment(path) {
   return hasNodeIgnoreComment(node);
 }
 
+const COMMENT = {
```

Comment:
I'd prefer `Comment` instead of `COMMENT`. That's the naming convention for enums in TS. E.g. https://github.com/microsoft/TypeScript/blob/5b2fba33349373240560b39c8b6779e3c0c29975/src/compiler/types.ts#L6337

Reply:
Probably, `Comment` is a bad name too. E.g. in the linked example it's `TransformFlags`, not just `Transform`.

Reply:
CommentType?

Reply:
"type" is bad because comment nodes already have the `type` property with a different meaning. I propose `CommentFlags`.

Reply:
How about `CommentCheckFlags`?

Reply:
I'm fine with that.

Reply:
As for enum members, there are two naming conventions:

1. TypeScript: `TransformFlags.ContainsJsx`
2. Java-like, also used in the browser DOM API: `Node.DOCUMENT_POSITION_PRECEDING`

Let's choose one of these. I personally like the first one.

Reply:
I like first one too.